src/components/InputControl/InputControl.js

```diff
@@ -1,6 +1,5 @@
-// TODO: not sure this component is neede - might as well be replaced with Bootstrap component
```

Comment:
Tbh I doubt this component is needed because we can just the one from Bootstrap. Might be a separate PR though